src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanicsConformingFractures.hpp

```diff
@@ -31,7 +31,7 @@ namespace singlePhasePoromechanicsConformingFracturesKernels
 {
 
 template< integer NUM_EQN, integer NUM_DOF >
-class ConnectorBasedAssemblyKernel : public singlePhaseFVMKernels::FaceBasedAssemblyKernel< NUM_EQN, NUM_DOF, SurfaceElementStencilWrapper >
+class ConnectorBasedAssemblyKernel : public singlePhaseFVMKernels::FluxComputeKernel< NUM_EQN, NUM_DOF, SurfaceElementStencilWrapper >
```

Comment:
I would have almost expected `ConnectorBasedAssemblyKernel` to be the base class for all `FluxAssemblyKernels`. I do like this name signficantly better than FaceBasedAssemblyKernel coz it's also a lot more general.